specification/ml/find_file_structure/MlFindFileStructureRequest.ts

```diff
@@ -0,0 +1,34 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { RequestBase } from '@_types/Base'
+
+/**
+ * Depreacted in 7.12
+ * see: https://www.elastic.co/guide/en/elasticsearch/reference/7.12/ml-find-file-structure.html
+ *
+ * @rest_spec_name ml.find_file_structure
```

Comment:
Should we track a deprecated API?
It was never in GA.

Reply:
left it as comment, we need to adjust these deprecations upstream in the spec. But, having the information is necessary for 7.x clients using the spec.